variants/XMC4200/config/XMC4200_Distance2Go/pins_arduino.h

```diff
@@ -0,0 +1,226 @@
+/*
```

Comment:
This file appears to have started from Sense2GoL file which has many errors in it I am working on an improved one for that